common/micc.js

```diff
@@ -95,52 +95,44 @@ function Micc(serverAddress) {
         }
     }
 
-    this.pickQueueConversation = function (conversationId, queueId) {
+    this.pickQueueConversation = function (conversationId, queueId, processResponse) {
         this.putQueueConversation(queueId, conversationId, `{
             conversationAction: 'Pick',
             id: '${conversationId}'
-            }`, function processResponse(responseData) {
-                console.log(`Response for 'Pick' for conversation ${conversationId}:  `, responseData);
-            });
+            }`, processResponse);
     }
 
-    this.AcceptConversation = function (conversationId, employeeId, isReplyAll) {
-        var tags = [{ key: 'emailAcceptType', value: isReplyAll ? "ReplyAll" : "Reply" }];
+    this.acceptConversation = function (conversationId, employeeId, tags, processResponse) {
         this.putEmployeeConversation(employeeId, conversationId, `{
             conversationAction: 'Accept',
             id: '${conversationId}',
-            tags: ${JSON.stringify(tags)}
-            }`, function processResponse(responseData) {
-                console.log(`Response for 'Accept' for conversation ${conversationId}:  `, responseData);
-            });
+            tags: ${tags}
```

Comment:
stringify was removed.  Was this deliberate?

As a matter of usage that other demos may leverage, I'd like to suggest this:  if tags MUST be an array, then it's appropriate for this function to accept the array and do the stringify here.  If the data type tags varies by context, then what you've done is the appropriate course.